<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataWidget</class>
 <widget class="QWidget" name="DataWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>431</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Light</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Arduino</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>301</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Light</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Arm MPU6050 sensor</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>70</y>
     <width>301</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Light</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Hand MPU6050 sensor</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>135</y>
     <width>61</width>
     <height>161</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Light</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Yaw
Pitch
Roll</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>135</y>
     <width>61</width>
     <height>161</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Light</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Yaw
Pitch
Roll</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView_2">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>140</y>
     <width>201</width>
     <height>111</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>140</y>
     <width>51</width>
     <height>151</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Light</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>-180

-180

-90</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>140</y>
     <width>51</width>
     <height>151</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Light</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>180

180

90</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>140</y>
     <width>51</width>
     <height>151</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Light</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>-180

-180

-90</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>140</y>
     <width>201</width>
     <height>111</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>140</y>
     <width>51</width>
     <height>151</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Light</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>180

180

90</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>140</y>
     <width>51</width>
     <height>151</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Light</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>140</y>
     <width>51</width>
     <height>151</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Light</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView_3">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>350</y>
     <width>51</width>
     <height>111</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>350</y>
     <width>31</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Light</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>345</y>
     <width>81</width>
     <height>161</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Light</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Thumb
Index
Middle</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>280</y>
     <width>301</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Light</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Hand photoresistors</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
